model/src/main/kotlin/CopyrightFinding.kt

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
commit: what effect on the file size in particular for large scans does this roughly have? 
maybe its worth mentioning in the commit message.